### ISSUE TITLE
Request: add SyncTrayzor to the Chocolatey repository

### ISSUE THREAD

**Issue by zilexa** [canton7/SyncTrayzor#189]
Please consider creating a choco package of Chocolatey. I now install all my non-Windows Store applications via choco and it is very easy. 
https://chocolatey.org

A Syncthing package already exists but it does not make sense to use that one as it simply installs the executable in the hidden programdata folder.

**Comment by canton7** [comment]
Someone emailed me about this the other day. My response (which has so far gone unanswered):

I've been considering a chocolatey package. Last time I looked at it, it seemed like we'd need four chocolatey packages: one each for x64 and x86, installed and portable. Do you know if this is still the case?

I'm definitely open to having a (or multiple) chocolatey packages. My preferred approach would be to let someone else figure out how to create it (someone who knows more about how they're supposed to work than I do), then I'll integrate that into the build process: so a chocolatey package is built by AppVeyor on each commit, and releasing to chocolatey becomes part of my normal release process.

I'd prefer to keep control of any accounts on other sites related to SyncTrayzor: while it's great to have someone else set that stuff up at the start, I'm very aware that over the course of a few years people lose interest or get caught up in other projects, and I need to be able to keep things going.

In summary, I'm definitely open to a chocolatey package (or multiple packages, as required). If you can figure out how to build it/them and write whatever powershell files need writing, I'll take them and integrate them into the build process, and figure out how to publish them from AppVeyor (I think that's supported).

**Comment by canton7** [comment]
Submitted for moderation

**Comment by canton7** [comment]
aand we're approved! https://chocolatey.org/packages/synctrayzor

**Comment by canton7** [comment]
Closing
